File: cluster/src/main/scala/com/oceanum/graph/GraphStatus.scala
package com.oceanum.graph

/**
 * @author chenmingkun
 * @date 2020/7/14
 */
object GraphStatus extends Enumeration {
  type value = Value
  val OFFLINE: GraphStatus.value = Value(1)
  val RUNNING: GraphStatus.value = Value(2)
  val SUCCESS: GraphStatus.value = Value(3)
  val EXCEPTION: GraphStatus.value = Value(4)
  val FAILED: GraphStatus.value = Value(5)
  val KILLED: GraphStatus.value = Value(6)
}

File: commons/src/main/java/com/oceanum/common/GraphStatus.java
package com.oceanum.common;

/**
 * @author chenmingkun
 * @date 2020/8/9
 */
public enum GraphStatus {
    OFFLINE(0),
    RUNNING(1),
    SUCCESS(2),
    EXCEPTION(3),
    FAILED(4),
    KILLED(5);

    int value;

    GraphStatus(int value) {
        this.value = value;
    }
}
